src/bloomd/networking.c

```diff
@@ -962,6 +1065,20 @@ static int set_client_sockopts(int client_fd) {
         return 1;
     }
 
+    // Set keep alive
+    int flag = 1;
+    if(setsockopt(client_fd, SOL_SOCKET, SO_KEEPALIVE, &flag, sizeof(int))) {
```

Comment:
Upon successful completion, setsockopt() shall return 0. Otherwise, -1 shall be returned and errno set to indicate the error.